src/Query/DateParser.ts

```diff
@@ -0,0 +1,8 @@
+import * as chrono from 'chrono-node';
+
+export class DateParser {
```

Comment:
how about making a TaskDate class and have that deal with all the date management across the plugin.

Reply:
That is a really nice idea. I will add it to my list, thank you.

Reply:
TaskDate class idea logged in #1104